internal/errors.go

```diff
@@ -45,14 +45,26 @@ func TranslateError(err error, isCloudCluster bool, op ...string) (string, bool)
 
 func TranslateClusterError(err error, operation string) (string, bool) {
 	var urlError *url.Error
+	isRestApiEnabledError := `Cannot access Hazelcast REST API.
```

Comment:
Let's always use upper case initialisms and `Err` instead of `Error`, so how about `restAPIErr`?

Reply:
Sure, thanks.